internal/config/5_normal_defaults.go

```diff
@@ -0,0 +1,41 @@
+//go:build !test && !integration
+// +build !test,!integration
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config/parser"
+	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
+)
+
+func init() {
+	if clowder.IsClowderEnabled() {
+		cfg := clowder.LoadedConfig
+		parser.Viper.Set("featureFlags.environment", "production")
+		parser.Viper.Set("database.host", cfg.Database.Hostname)
+		parser.Viper.Set("database.port", cfg.Database.Port)
+		parser.Viper.Set("database.user", cfg.Database.Username)
+		parser.Viper.Set("database.password", cfg.Database.Password)
+		parser.Viper.Set("database.name", cfg.Database.Name)
+		parser.Viper.Set("prometheus.port", cfg.MetricsPort)
+		parser.Viper.Set("prometheus.path", cfg.MetricsPath)
+		if endpoint, ok := clowder.DependencyEndpoints["sources-api"]["svc"]; ok {
+			parser.Viper.Set("restEndpoints.sources.url", fmt.Sprintf("http://%s:%d/api/sources/v3.1", endpoint.Hostname, endpoint.Port))
+		}
+	} else {
+		parser.Viper.SetDefault("featureFlags.environment", "development")
+
+		// load from main subdirectory
+		parser.Viper.AddConfigPath("./configs")
+		// load from working directory for integration tests
+		parser.Viper.AddConfigPath(".")
+		parser.Viper.SetConfigName("local")
+
+		err := parser.Viper.ReadInConfig()
+		if err != nil {
+			fmt.Println("Could not read configs/local.yaml, continuing with defaults")
+		}
+	}
+}
```

Comment:
On the other hand, these are not just defaults, so maybe to rename this back? Not sure.

Reply:
yeah, the others are defaults, but this one we use to set variables from clowder, so those are final IMHO, so others default and this one main or smthing?

Reply:
shall we move these into main config? :thinking:

Reply:
I don't know, it contains both defaults and both production code.